packages/eslint-plugin-fxa/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "eslint-plugin-fxa",
+  "version": "2.0.2",
+  "description": "eslint plugin for Firefox Accounts",
+  "main": "lib/index.js",
+  "scripts": {
+    "test": "mocha"
+  },
+  "dependencies": {
+    "eslint": "6.6.0",
```

Comment:
I've noticed that our eslint versions are all over the place. Should we consolidate one eslint versions? This is probably a follow up task...

Reply:
I like what you did with moving the the fxa eslint plugin to the root package.json. Maybe we do the same thing?

Reply:
Sure, I can file an issue for this.

Reply:
Obligatory <kbd>grep</kbd>:

```sh
git grep '"eslint":' packages/**/package.json | cat

packages/123done/package.json:    "eslint": "^7.32.0",
packages/browserid-verifier/package.json:    "eslint": "^7.32.0",
packages/eslint-plugin-fxa/package.json:    "eslint": "6.6.0",
packages/eslint-plugin-fxa/package.json:    "eslint": ">=6.6.0"
packages/fortress/package.json:    "eslint": "^7.32.0",
packages/fxa-admin-panel/package.json:    "eslint": "^7.32.0",
packages/fxa-admin-server/package.json:    "eslint": "^8.18.0",
packages/fxa-auth-server/package.json:    "eslint": "^7.32.0",
packages/fxa-content-server/package.json:    "eslint": "^7.32.0",
packages/fxa-customs-server/package.json:    "eslint": "^7.32.0",
packages/fxa-event-broker/package.json:    "eslint": "^7.32.0",
packages/fxa-geodb/package.json:    "eslint": "^8.19.0",
packages/fxa-graphql-api/package.json:    "eslint": "^7.32.0",
packages/fxa-payments-server/package.json:    "eslint": "^7.32.0",
packages/fxa-profile-server/package.json:    "eslint": "^8.18.0",
packages/fxa-react/package.json:    "eslint": "^7.32.0",
packages/fxa-settings/package.json:    "eslint": "^7.32.0",
packages/fxa-shared/package.json:    "eslint": "^7.32.0",
```

Or if you prefer uniques:
```sh
git grep -h '"eslint":' packages/**/package.json | sort | uniq

    "eslint": "6.6.0",
    "eslint": ">=6.6.0"
    "eslint": "^7.32.0",
    "eslint": "^8.18.0",
    "eslint": "^8.19.0",
```